static/third/marked/lib/marked.js

```diff
@@ -850,7 +865,6 @@ InlineLexer.prototype.userGravatar = function (email) {
   return this.options.avatarHandler(email);
 };
 
-
```

Comment:
Was that newline intentional? I'm asking because with it deleted, now all `InlineLexer` blocks have a blank line between them.